content/tutorial-paychan.md

```diff
@@ -0,0 +1,489 @@
+                "status": "success"
+            }
+        }
+
+
+4. **The payer transmits a claim to the payee as payment for goods or services.**
+
+    This communication happens "off-ledger" in any communication system the payer and payee can agree to. It's recommended that the two parties use secure communications for this, but it's not strictly necessary. Only the payer or payee of a channel can redeem claims against that channel.
+
+    The exact format of the claim is not important as long as it communicates the following information:
+
+    | Field                   | Example                                            |
+    |:------------------------|:---------------------------------------------------|
+    | Channel ID              | `5DB01B7FFED6B67E6B0414DED11E051D2EE2B7619CE0EAA6286D67A3A4D5BDB3` |
+    | Amount of XRP, in drops | `1000000`                                          |
+    | Signature               | `304402204EF0AFB78AC23ED1C472E74F4299C0C21F1B21D07EFC0A3838A420F76D783A400220154FB11B6F54320666E4C36CA7F686C16A3A0456800BBC43746F34AF50290064` |
+
+    The payee also needs to know the Public Key associated with the channel, which is the same throughout the channel's life.
+
+5. **The payee verifies the claims and provides the goods or services.**
+
+    You can verify claims using the `channel_verify` API method. The payee should confirm that the amount of the claim is equal or greater than the total price of goods and services provided. (Since the amount is cumulative, this is the total price of all goods and services purchased so far.)
+
+    Example of using `channel_verify` with the JSON-RPC API:
+
+    Request:
+
+        POST http://localhost:5005/
+        Content-Type: application/json
+
+        {
+            "method": "channel_verify",
+            "params": [{
+                "channel_id": "5DB01B7FFED6B67E6B0414DED11E051D2EE2B7619CE0EAA6286D67A3A4D5BDB3",
+                "signature": "304402204EF0AFB78AC23ED1C472E74F4299C0C21F1B21D07EFC0A3838A420F76D783A400220154FB11B6F54320666E4C36CA7F686C16A3A0456800BBC43746F34AF50290064",
+                "public_key": "aB44YfzW24VDEJQ2UuLPV2PvqcPCSoLnL7y5M1EzhdW4LnK5xMS3",
+                "amount": "1000000"
+            }]
+        }
+
+    Response:
+
+        200 OK
+
+        {
+            "result": {
+                "signature_verified":true,
+                "status":"success"
+            }
+        }
+
+    If the response shows `"signature_verified": true` then the claim is genuine. At this point, the payee has not yet received the XRP, but can be certain that he or she _can_ redeem the XRP as long as the transaction to do so is processed before the channel expires. ***Rome's question: Is there anything else the payee should check, or any tricks the payer could use to make a "valid" signature for a balance that couldn't actually be redeemed?***
```

Comment:
`channel_verify` shows the claim is genuine, as you state. However note it does _not_ verify the channel has enough funds available to honor the claim. The payee may want to check that the channel has enough funds to honor the claim.